proto/tabletmanagerdata.proto

```diff
@@ -71,6 +71,22 @@ message SchemaChangeResult {
   SchemaDefinition after_schema = 2;
 }
 
+// TabletControl controls tablet behavior regarding routing and is partly used
+// to determine if we can serve data
+message TabletControl {
+  // Tablet type
+  topodata.TabletType tablet_type = 1;
+
+  repeated string cells = 2;
+
+  // should query service be enabled on this shard
+  bool query_service = 3;
+
+  repeated string denylist_tables = 4;
+
+  bool frozen = 5;
+}
```

Comment:
From the RPC, it seems like the idea is to move some information from the shard level to the tablet level.
Assuming that to be the case, I'm not sure it makes sense to have all of these fields. Some of them made sense when the tablet control was at the shard level by tablet_type.

Reply:
Reading #7688 makes this much clearer. Because the list of tablet controls is replicated across all tablets, it makes sense that tablet_type and cell are part of the message.

Reply:
I think the only change needed is to replace `cells` with `cell` as described in the design.